lib/tasks/data_hygiene/inconsistent_redirect_finder.rb

```diff
@@ -0,0 +1,16 @@
+class InconsistentRedirectFinder
+  attr_reader :content_items
+
+  def initialize(content_items)
+    @content_items = content_items
+  end
+
+  def items_with_inconsistent_redirects
+    content_items.select do |content_item|
+      next if content_item.format == "redirect"
+
+      route = Rails.application.router_api.get_route(content_item.base_path)
+      route && route.handler == "redirect" && !route.redirect_to.to_s.include?('#')
```

Comment:
I put that in to filter the policies that redirected to anchor tags:

```
/government/policies/helping-developing-countries-economies-to-grow/supporting-pages/the-investment-facility-for-utilising-uk-specialist-expertise-ifuse
```

redirects to:

```
/government/publications/2010-to-2015-government-policy-economic-growth-in-developing-countries/2010-to-2015-government-policy-economic-growth-in-developing-countries#appendix-7-the-investment-facility-for-utilising-uk-specialist-expertise-ifuse
```

But I guess that should have gone into the rake task to find the policies as it's specific to that use case. Removed in https://github.com/alphagov/content-store/commit/2160bfb0e3931e1525b105552311d9fe9d196167.